### PR TITLE
Feature b new

### PR DESCRIPTION
**Pull request by patrodriguez108**
@Bontgoy Hi Tyler. So my work in this resulted in some convolutions. I ended up creating and pushing about three more branches than necessary, and two of them still have not been deleted.

The first branch I made was feature-b and I pushed and pulled that one just fine. I however was not sure if I properly merged it with my master branch. I kept going back to the git-defintions.md file in the master branch and none of my answers were there. I ended figuring how to merge feature-b with the master branch through GitHub; however once I did so I found that in both the master and in feature-b none of my answers were appearing in git-definitions.md

I ended up going back to my terminal and making feature-branch-new. This one I pushed and pulled, and successfully merged it with my master branch. However I went to the master branch and I still saw that none of my answers were appearing, although they were appearing in the code view. I realized that none of my answers were appearing because I did not indent my answers. They looked like this:

*Question
Answer
*Question
Answer

So I hit the edit button and made the changes within GitHub. I made it so it looked like:

*Question

Answer

*Question

Answer

Instead of merging these changes right into my master branch, I made a separate branch that named itself patch or something similar. Finally I went to git-definitions.md and saw that my answers appeared. I went ahead and merged this branch with the master branch.

Because I did not make that change on my local computer, in submitting this pull request I made the indentation change.

Tl;dr I have two branches on GitHub (feature-b and new-feature-b) that I want to delete, but I do not know how to.

### CODE REVIEW

**Comment by Bontgoy** [comment]
@patrodriguez108 This is the wrong repository to make this pull request to.  Please close this PR and redo the pull request process.  This time in the `base/fork` drop down select your repository.  It is the one that you forked rom DBC-Tracks.  Then we can work on merging this successfully.

Reach out on Slack if you have any questions :)
